templates/verifier_groth.sol

```diff
@@ -182,11 +182,13 @@ contract Verifier {
         <%vk_ic_pts%>
     }
     function verify(uint[] memory input, Proof memory proof) internal view returns (uint) {
+        uint256 snark_scalar_field = 21888242871839275222246405745257275088548364400416034343698204186575808495617;
         VerifyingKey memory vk = verifyingKey();
         require(input.length + 1 == vk.IC.length,"verifier-bad-input");
         // Compute the linear combination vk_x
         Pairing.G1Point memory vk_x = Pairing.G1Point(0, 0);
         for (uint i = 0; i < input.length; i++)
```

Comment:
You need curly braces here.

Reply:
I guess my tests were bad? Let me check.